src/engine/idTable/IdTable.h

```diff
@@ -783,9 +783,8 @@ class IdTable {
     const size_t numInserted = indices.size();
     if (numInserted == 0) return;
 
-    const size_t srcSize = table.size();
     AD_EXPENSIVE_CHECK(ql::ranges::all_of(
-        indices, [srcSize](size_t idx) { return idx < srcSize; }));
+        indices, [&table](size_t idx) { return idx < table.size(); }));
```

Comment:
Does this change anything in behaviour?

Reply:
Before this change, there were very many `unused` warnings (for all compilations where `AD_EXPENSIVE_CHECK` is noop). Since the function is both `const` and constant-time, I don't see a drawback